trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+
+# create TrimmerDecorator class
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = super
+    return name[0, 10] if name.length > 10
```

Comment:
- [ ] @sheezu900 As previously highlighted we need to use a nameable variable containing the object of the new person and its attribute then from it get the name, also your code returns a name of 11 characters instead of 10 eg ``@nameable.correct_name. length > 10? @nameable.correct_name[0..9] : @nameable.correct_name``